OADP-3353: Multiple BSL are supported (#70623)
openshift/openshift-docs: modules/about-installing-oadp-on-multiple-namespaces.adoc

You install each instance of OADP as specified by the per-platform procedures contained in this document with the following additional requirements:

* All deployments of OADP on the same cluster must be the same version, for example, 1.1.4. Installing different versions of OADP on the same cluster is *not* supported.

> * Each individual deployment of OADP must have a unique set of credentials and at least one `BackupStorageLocation` configuration. Multiple `BackupStorageLocation` configurations within the same namespace are also possible.

Reviewer: Suggest to make it active voice

```suggestion
* Each individual deployment of OADP must have a unique set of credentials and at least one `BackupStorageLocation` configuration. You can also use multiple `BackupStorageLocation` configurations within the same namespace.
```
